src/Core/Data/Contexts/AppContext.hpp

```
@@ -6,9 +6,16 @@
 #include <cstdint>
 #include <condition_variable>
 
+#include <imgui/imgui.h>
+
 
 // General application context
 struct AppContext {
+    struct Window {
+        float dpiScale = 0.0f;
```

Clang (14.0.6)

[warning] floating point literal has suffix 'f', which is not uppercase

(readability-uppercase-literal-suffix)